src/site/content/en/blog/accent-color/index.md

```diff
@@ -0,0 +1,180 @@
+---
+layout: post
+title: CSS `accent-color`
+subhead: Bring your brand color to built-in HTML form inputs with one line of code.
+authors:
+  - adamargyle
```

Comment:
would it be cool if i'm an author too...?

Reply:
of course!

Reply:
https://github.com/GoogleChrome/web.dev/pull/5953

Reply:
I approved the PR adding you, you'll need to merge the main branch into this one I think, otherwise, the build will complain.

Reply:
![Screen Shot 2021-08-10 at 9 07 13 AM](https://user-images.githubusercontent.com/1134620/128896936-3272d45a-d4a8-4f8d-9994-fe62ec92aa3f.png)

Reply:
github is being wierd today and the commits say they pushed but arent reflected in this PR yet